radixdlt/src/main/java/org/radix/api/services/AtomsService.java

```diff
@@ -155,6 +121,42 @@ public AtomsService(LedgerEntryStore store, SubmissionControl submissionControl,
 		});
 	}
 
+	public void start() {
+		io.reactivex.rxjava3.disposables.Disposable lastStoredAtomDisposable = engineStore.lastStoredAtom()
```

Comment:
Less use of `Events` is nice.

Hopefully we can sort out the name clash on `Disposable` shortly.